app/src/main/kotlin/com/saltedge/authenticator/features/actions/SubmitActionFragment.kt

```diff
@@ -65,81 +66,89 @@ class SubmitActionFragment : BaseFragment(), SubmitActionContract.View, View.OnC
             titleResId = R.string.action_new_action_title,
             backActionImageResId = R.drawable.ic_appbar_action_close
         )
-        return inflater.inflate(R.layout.fragment_submit_action, container, false)
+        binding = DataBindingUtil.inflate(
+            inflater,
+            R.layout.fragment_submit_action,
+            container,
+            false
+        )
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this
+        return binding.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         completeView?.setOnClickListener(this)
```

Comment:
you listen clicks only from single view. use lambda instead of global implementation